src/frontend/react/docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: "3.8"
```

Comment:
自分のdockerが古いのかもしれませんが、unsupportedなバージョンだと言われてしまいました。
```
$ docker --version
Docker version 19.03.12, build 48a66213fe
$ docker-compose up -d
ERROR: Version in "./docker-compose.yaml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
```

19.03.* なので、大丈夫そうなのですが・・・。
差し支えなければ、念のため"3.7"に落とした方が良さそうです。

Reply:
composeのバージョンを3.7に落とします